recycleIT-frontend/src/Components/Athorization/Authorization.tsx

```diff
@@ -254,9 +302,14 @@ const Authorization = () => {
                         }
                     </Button>
                 </Typography>
+                {
+                    error ? <AlertMessageBox error text={errorMessage}/> : null
```

Comment:
can be replaced with `{ error && <AlertMessageBox error text={errorMessage}/> }`